Makefile

```diff
@@ -12,8 +18,16 @@ clean: $(apps) $(examples)	## Clean build for all applications and examples
 
 distclean: PF_TARGET = distclean
 distclean: $(apps) $(examples)	## Clean build and configuration for all applications
+	@cd $(riot_path) && (quilt pop -a || true) && rm patches .pc -rf
```

Comment:
I don't know what is quilt, but it has to be installed first, could you update the README?

Reply:
done